src/modules/header/header.tmpl.js

```diff
@@ -0,0 +1,29 @@
+import {PROFILE, CHAT, authorizedPaths} from '../../router/namePath'
+
+const context = {
+  isAuthorizedPaths: authorizedPaths.includes(location.pathname)  
+};
+
+const personIcon = new URL('../../img/person.svg', import.meta.url);
+const chatIcon = new URL('../../img/chat.svg', import.meta.url);
+let icon = location.pathname === CHAT ? personIcon : chatIcon
+let href = location.pathname === CHAT ? PROFILE : CHAT
```

Comment:
```suggestion
const icon = location.pathname === CHAT ? personIcon : chatIcon
const href = location.pathname === CHAT ? PROFILE : CHAT
```

переменные не изменяются